docs/src/data/power.js

```diff
@@ -71,6 +71,18 @@ export const underglowPower = {
   ledOff: 460, // Quiescent current of a WS2812B
 };
 
+export const backlightLEDs = {
+  White: 3.2,
+  Blue: 3.0,
+  Green: 2.2,
+  Yellow: 2.1,
+  Red: 1.8,
+};
+
+export const backlightPower = {
+  pwmPower: 500, // Estimated power consumption of PWM module
```

Comment:
I'll test this soon and get some real-world numbers for you.

Reply:
Just tested this. These results are specific to nrf52840. At 100% and 0% brightness, PWM uses around 0-2uA (negligible basically and probably off due to a smart LED implementation). At anything in between, it's around 510uA. Not sure if you want to add in the logic to check if brightness is 100% or 0%.